### PR TITLE
busybox: allow changing root password

### PR DESCRIPTION
**Pull request by chewitt**
I've been testing @lrusak work on this for a bit and haven't seen any issues so it's time for us to be a big grown up distro and allow users to change (and forget) passwords. Some UX work is required to expose it in the LE settings add-on and first-run wizard, but those will be worked on separately. This PR supersedes the original idea in #1354 from @vpeter4 which we left open as a placeholder to ensure password change wasn't forgotten about.

### CODE REVIEW

**Comment by lrusak** [comment]
The patch needs fixing

**Comment by chewitt** [comment]
fixed.. hopefully

File: packages/sysutils/busybox/package.mk [hunk 1]
```diff
@@ -156,7 +156,7 @@ makeinstall_target() {
     cp $PKG_DIR/scripts/createlog $INSTALL/usr/bin/
     cp $PKG_DIR/scripts/lsb_release $INSTALL/usr/bin/
     cp $PKG_DIR/scripts/apt-get $INSTALL/usr/bin/
-    cp $PKG_DIR/scripts/passwd $INSTALL/usr/bin/
+    #cp $PKG_DIR/scripts/passwd $INSTALL/usr/bin/
```
Comment (CvH) [inline packages/sysutils/busybox/package.mk]: should be deleted instead of commented out

Comment (chewitt) [reply packages/sysutils/busybox/package.mk]: done

**Comment by Raybuntu** [comment]
Thanks all. Especially @vpeter4.
I have been using this for months in my builds.

**Pull request merged**